src/api/parser/jest.setup.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const result = require('dotenv').config({
```

Comment:
I think this has been done in the base config (cc @HyperTHD).

Reply:
Yup, this file is not needed anymore and can be removed